airbyte-integrations/bases/airbyte-protocol/build.gradle

```diff
@@ -2,11 +2,13 @@ project.ext.pyModule = 'airbyte_protocol'
 apply from: rootProject.file('tools/gradle/commons/integrations/python.gradle')
 apply from: rootProject.file('tools/gradle/commons/integrations/image.gradle')
 
-
-task generateProtocolClassFiles(type: Exec) {
+task generateProtocolClassFilesWithoutLicense(type: Exec) {
     environment 'ROOT_DIR', rootDir.absolutePath
     commandLine 'bin/generate-protocol-files.sh'
     dependsOn ':airbyte-integrations:bases:airbyte-protocol:code-generator:buildImage'
 }
 
+task generateProtocolClassFiles(dependsOn: [generateProtocolClassFilesWithoutLicense, rootProject.spotlessPythonApply])
```

Comment:
is this just saying that `assemble.dependsOn [generateWithoutLicense, spotlessPythonApply]` and that spotlessApply must run after generateWithoutLicense? If so I think that's more straightforward to understand.

Reply:
This is also exposing a task that independently can generate protocol class files with licenses. If we only wanted to generate files as part of `./gradlew build` I think the approach you're describing is fine, but I would like to have a command `generateProtocolClassFiles` which is completely independent. 

I could merge the depends into the earlier task though, which is probably just an improvement. I'll do that now.

Reply:
Actually, after doing that I reverted. It feels weird to reorder a direct dependency instead of specifying the order of tasks for equal-level dependencies.